src/wp-admin/edit-form-blocks.php

```diff
@@ -72,6 +72,9 @@ static function( $classes ) {
 	sprintf( '%s/autosaves?context=edit', $rest_path ),
 	'/wp/v2/settings',
 	array( '/wp/v2/settings', 'OPTIONS' ),
+	'/wp/v2/media?context=edit&per_page=1&orderBy=date&media_type=image',
+	'/wp/v2/media?context=edit&per_page=1&orderBy=date&media_type=video',
+	'/wp/v2/media?context=edit&per_page=1&orderBy=date&media_type=audio',
```

Comment:
I'm not sure if it's preferred in this format:
```
add_query_arg(
	array(
		'context'    => 'edit',
		'per_page'   => 1,
		'orderBy'    => 'date',
		'media_type' => 'image',
	),
	rest_get_route_for_post_type_items( 'attachment' )
),
```

Reply:
Yeah this would be better.